### DISCUSSION TITLE
My `readme.org` is shown as `README.md`

### DISCUSSION THREAD

**Discussion by rwp0** [community/community#67528] [Mobile] [answered]
![Screenshot_20230919_003639_GitHub.jpg](https://github.com/community/community/assets/68062695/ef8c2f04-3356-4bbe-bc77-cad8dd2d02fe)

My readme is org[^1] and all lowercase.

This seems to be a bug in GitHub Mobile.

https://github.com/rwp0/rwp0/blob/master/readme.org

[^1]: Formatted in Org-mode and not Markdown (.md)

**Comment by stevepopovich** [comment]
Hi @rwp0 , sorry I am a bit confused here. What is the behavior you expect here?

**Reply by rwp0** [reply]
It calls my `readme.org` file `README.md`
I expect it to call my file by its real name, and not assume it's formatted in Markdown (since GitHub supports non-Markdown README's too).

**Reply by stevepopovich** [reply, marked as answer]
Gotcha, ok, I will pass this onto the team internally 👍🏼

**Comment by rwp0** [comment]
Thanks GitHub!

Today I got the below news when updating my GitHub mobile app on Android.

![image](https://github.com/community/community/assets/68062695/f41260d5-93a7-47de-91a3-ab2cfca96c3a)

**Reply by stevepopovich** [reply]
No problem 👍🏼 Thanks again for reporting!

**Reply by rwp0** [reply]
Just for the record, that's how it's displayed now:

![image](https://github.com/community/community/assets/68062695/f5357564-5fff-44db-872a-3167c849c328)
